### DISCUSSION TITLE
Welcome to the study group general and random discussions!

### DISCUSSION THREAD

**Discussion by dprada** [uibcdf/Study_Group_Pharmacophores#1] [General]
## 👋 Welcome!
  We’re using this place not to talk about scientific topics, but to discuss the dynamics of the discussion group. Then, if you have a suggestion or a comment on how this repo should work, feel free to do it here. 🤓

**Comment by dprada** [comment]
@Daniel-Ibarrola I'm not sure how you are editing the file `bibliography.tex`. I hope you are not adding the bibtext fields by hand. Here a couple of tips:

- Check that getting the bibtex entry of the google scholar results is easy. Below every paper in the resultant list, there is a bold quotation mark (it says `cite` if you put the mouse there). Click this symbol and a new small window unfolds with the word "bibtex" at the bottom. There you can find a complete bibtex entry you can copy-paste.
- If the paper is in Mendeley, this software can export a paper or a list of papers to bibtex. With Mendeley Desktop, right-click the name of the paper and choose 'export'. There you have it.

**Reply by Daniel-Ibarrola** [reply]
Ok, I will look into it and correct the bad ones.

**Comment by dprada** [comment]
By the way, I had to make this repo public. If we keep it private some interesting functions are disabled. If any of you have a special interest in turning it back to private, say it here.

**Comment by dprada** [comment]
@Daniel-Ibarrola, I don't know why but your commits do not appear in the section 'Insights-Contributors'. They do in 'Insights-Pulse', but your name together with a plot commits.vs.time should be in 'Insights-Contributors'. Have a look at [the contributors of Yank](https://github.com/choderalab/yank/graphs/contributors) to understand what I mean. You could think this is not important now. But whenever we start to implement the library, this has to be solved. This is the best way to keep track of your contribution. Imagine you wrote 80% of the lines of code... this number must be public -for all to know that you were the main developer-. Otherwise, it is unfair.

[Here](https://docs.github.com/en/github/visualizing-repository-data-with-graphs/viewing-a-projects-contributors#troubleshooting-contributors) I found a possible source of error. Check it out.

**Reply by Daniel-Ibarrola** [reply]
I checked the link but that doesn't seem to be the issue beacuse my commits are in the default branch and I'm supposed to be the top 2 contributor. Maybe it has something to do with my account but I already checked that is the same email.  I'm going to keep looking into that.

**Reply by dprada** [reply]
I'll also see what could be the reason why you are not on the contributors' list.

**Reply by dprada** [reply]
I am glad this was fixed having a look at the instructions to set a traceable email address in your commits as indicated [here]( https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address).
